hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson

```diff
@@ -39,6 +39,14 @@
 
   en_build_modes: ["{tool}_crypto_dpi_prince_build_opts"]
 
+  build_modes: [
+    {
+      name: default
+      // Re-generate otp_ctrl_part_pkg with random seed.
+      pre_build_cmds: ["cd /${proj_root} && ./util/design/gen-otp-mmap.py --seed ${seed}"]
```

Comment:
o crap,  i didnt know we could do this.
Although do we end up with an issue where the design tree might get modified, and we'll just have to be careful not to accidentally commit them?

Reply:
That is a good point, this actually should only change the `src` files from the copied build directory, i believe under - `default/src`. It should not actually change the github repo :)

Reply:
o that's even better.

Reply:
To have this applied to the nightly regressions by default (which is what we want), you could do this instead:
```
  regressions: [
    {
      name: nightly
      pre_build_cmds: [...]
    }
  ]
```

Reply:
Thanks @sriyerg, that sounds great. Do you mind helping me understand these two questions:

1). If we added to nightly only, when we run `-i otp_ctrl_dai_errors`, it won't apply the pre_build_cmds. Is this correct? If so, I think it will be hard to reproduce nightly failures.

2). Do you remember during the meeting, did we mention we also want to randomize the seeds in top-level?

Thanks again!
CIndy

Reply:
> o crap, i didnt know we could do this. Although do we end up with an issue where the design tree might get modified, and we'll just have to be careful not to accidentally commit them?

Yeah I added this capability in dvsim [recently](https://github.com/lowRISC/opentitan/pull/11661). One of the reasons why I was suggesting not maintaining the rand const pkgs (rather have them generated on the fly instead) in our chatroom.

Reply:
Actually -this does modify the in-tree sources - so we need to be careful to avoid checking them in.

Reply:
Oh you are right @sriyerg, yeah I did not see the change in my repo because it was modified in the cloud.. Thanks for pointing that out. @tjaychen I think Sri is right - it does modify the in-tree sources.